docs: add MarkText to Vega-Lite ecosystem (#8283)
vega/vega-lite: site/ecosystem.md

- [Saite](https://github.com/jsa-aerial/saite) Interactive exploratory graphics and ad hoc visualization application for Clojure(Script). Built on top of [Hanami](https://github.com/jsa-aerial/hanami).
- [Datablocks](https://datablocks.pro), a node-based editor for exploring, analyzing and transforming data without code.
- [Rath](https://github.com/Kanaries/Rath) An augmented analysis tool including auto-EDA, pattern discovery, multi-dimensional visualization recommendation, and interactive dashboards generation.

> - [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite. 

Reviewer: ```suggestion
- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite.
```

> - [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports vega-lite diagram. 

Reviewer: ```suggestion
- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite. 
```

> - [MarkText](https://github.com/marktext/marktext): A simple and elegant open-source markdown editor that focused on speed and usability.

Reviewer: ```suggestion
- [MarkText](https://github.com/marktext/marktext): A markdown editor that focuses on speed and usability.
```

Author: Oh, that just referenced from MarkText description. 
I just an user not the developer. 
I think we can simply use markdown editor as the description if we need more objective.
